src/main/webapp/app/entities/companion-diagnostic-device/companion-diagnostic-device.tsx

```diff
@@ -25,6 +27,35 @@ export const CompanionDiagnosticDevice = (props: ICompanionDiagnosticDeviceProps
 
   const { match } = props;
 
+  const columns: Column<ICompanionDiagnosticDevice>[] = [
+    { id: 'id', accessor: 'id', Header: 'ID' },
```

Comment:
Do we need to show the id if we don't allow clicking through the id to the device page like we used to do?

Reply:
We can omit that information. Also, just realized that for react-table columns, we don't need to `id` field if `accessor` is a string.